src/plugins/navigation/public/top_nav_menu/top_nav_menu_item.tsx

```diff
@@ -22,6 +22,19 @@ export function TopNavMenuItem(props: TopNavMenuData) {
     return val!;
   }
 
+  function getButtonContainer() {
+    if (props.badge) {
+      return (
+        <div className="kbnTopNavMenu__betaBadgeWrapper">
+          <EuiBetaBadge className="kbnTopNavMenu__betaBadgeItem" {...props.badge} size="s" />
+          {upperFirst(props.label || props.id!)}
+        </div>
```

Comment:
Consider changing this wrapping `div` to a React fragment.

```suggestion
        <>
          <EuiBetaBadge className="kbnTopNavMenu__betaBadgeItem" {...props.badge} size="s" />
          {upperFirst(props.label || props.id!)}
        </>
```